### PR TITLE
Add setup db script

### PR DESCRIPTION
**Pull request by dplumb94**
Signed-off-by: Darian Plumb <dplumb@bitwise.io>

### CODE REVIEW

File: bin/setup_db [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        print('Database already exists:', name)
+
+    finally:
+        conn.close()
+        print('Connection closed')
+
+
+if __name__ == '__main__':
+    opts = parse_args(sys.argv[1:])
+    setup_db(opts.host, opts.port, opts.name)
```
Comment (delventhalz) [inline bin/setup_db]: Missing blank line

File: docker-compose-installed.yaml [hunk 2]
```diff
@@ -58,7 +58,11 @@ services:
     depends_on:
       - rethink
       - validator
-    command: echo "No ledger sync command set"
+    command: |
+      bash -c "
+        sleep 1 &&
+        bin/setup_db --host rethink
```
Comment (delventhalz) [inline docker-compose-installed.yaml]: add `--port 28042`

File: docker-compose.yaml [hunk 3]
```diff
@@ -62,7 +62,11 @@ services:
     depends_on:
       - rethink
       - validator
-    command: echo "No ledger sync command set"
+    command: |
+      bash -c "
+        sleep 1 &&
+        bin/setup_db --host rethink
```
Comment (delventhalz) [inline docker-compose.yaml]: Add `--port 28042`

File: bin/setup_db [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        print('Creating table: accounts')
+        r.db(name).table_create('accounts').run(conn)
+        r.db(name).table('accounts').index_create('account_id').run(conn)
+
+        print('Creating table: assets')
+        r.db(name).table_create('assets').run(conn)
+        r.db(name).table('assets').index_create('asset_id').run(conn)
+
+        print('Creating table: auth')
+        r.db(name).table_create('auth', primary_key='public_key').run(conn)
```
Comment (delventhalz) [inline bin/setup_db:55]: Below this line, add email as a secondary index:
```
r.db(name).table('auth').index_create('email').run(conn)
```

**Review by delventhalz — CHANGES_REQUESTED** [review]

File: bin/setup_db (hunk 1 above)
Comment (dplumb94) [reply bin/setup_db]: fixed

File: bin/setup_db (hunk 4 above)
Comment (dplumb94) [reply bin/setup_db:55]: added

**Review by delventhalz — APPROVED** [review]

**Review by boydjohnson — APPROVED** [review]

**Pull request merged**